assets/css/main.css

```diff
@@ -14,3 +14,4 @@
 @import "sections/form";
 
 @import "pages/home";
+@import "tasks/_global";
```

Comment:
I realise that  "tasks/_global" should be with out low dash: "tasks/global". Could you change it?

Reply:
Low dash deleted. Please review and merge as the task is acomplished. Maybe you might spread the news on slack? I think the final decision regarding dash/no dash issue in global.css would be important to everyone.